scripts/windows/install_msys2.bat

```diff
@@ -0,0 +1,66 @@
+@echo off
+:: Pass -v as first argument to enable debug echo
+:: Script to download and install Msys2 from %MSYS2_URL%
```

Comment:
Installing MSYS2 is quite complex and does not have an installer, so we need our own script to do it. It could be very helpful for users too